PENDING_RELEASE_NOTES.md

```diff
@@ -4,5 +4,6 @@ _Date TBD_
 
 ### New features
 * Updated security for edit/split/merge of developers, products, and versions
+* Updated product and version activity metadata to parse activity for merges and splits; better parsing of product and developer names if they have been deleted.
```

Comment:
Is this a backwards compatibility breaking change?

Reply:
No I don't think so.. the format of the request and response is the same for the activity metadata endpoints

Reply:
Oh! I see. You're using the single vs. list to determine which one to pull data from. Got it. I was half reading the code and late for a meeting at the same time :)